linera-core/src/data_types.rs

```diff
@@ -310,6 +321,13 @@ impl ChainInfoResponse {
 
 impl BcsSignable<'_> for ChainInfo {}
 
+/// Request for downloading certificates by height range.
+#[derive(Debug, Clone)]
+pub struct CertificatesByHeightRequest {
+    pub chain_id: ChainId,
+    pub heights: Vec<BlockHeight>,
+}
+
```

Comment:
I wonder if it would be better to have a single ‘get certificates’ endpoint with a sophisticated request type with various optional filters for the certificates.